ScienceJournal/Sensors/MagnetometerSensor.swift

```diff
@@ -64,12 +65,44 @@ class MagnetometerSensor: MotionSensor {
     guard let magneticField =
         MagnetometerSensor.motionManager.deviceMotion?.magneticField.field else { return }
 
+    guard isValid(magneticFieldReading: magneticField) else {
+      return
+    }
     // The magnetic strength is the square root of the sum of the squares of the values in X, Y
     // and Z.
-    let magneticStrength =
+    var magneticStrength =
         sqrt(pow(magneticField.x, 2) + pow(magneticField.y, 2) + pow(magneticField.z, 2))
+
+    // Typical white dwarf stars have a magnetic field of around 100 Tesla (100,000,000 µT), so
+    // let's cap our values there.
+    // Note: not a good idea to get close enough to measure a white dwarf star.
```

Comment:
Nit: "Note: Not a good"... (cap the N on Not)

Reply:
Done